pkg/cmd/eventlistener/describe.go

```diff
@@ -179,6 +182,25 @@ func describeEventListenerOutput(w io.Writer, p cli.Params, f *cliopts.PrintFlag
 	return printer.PrintObject(w, el, f)
 }
 
+func describeEventListenerOutputURL(w io.Writer, p cli.Params, name string) error {
+	cs, err := p.Clients()
+	if err != nil {
+		return err
+	}
+
+	el, err := cs.Triggers.TriggersV1alpha1().EventListeners(p.Namespace()).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if (getURL(*el)) == "" {
```

Comment:
You can use fromat package here,  as you used in list, so need for an if case here.